nmstatectl/nmstatectl.py

```diff
@@ -86,7 +86,7 @@ def setup_subcommand_set(subparsers):
     parser_set = subparsers.add_parser('set', help='Set network state')
     parser_set.add_argument('file', help='File containing desired state. '
```

Comment:
Now it is `File(s)`

Reply:
The help output is fine IMHO, since it shows that file occurs more than once:
```
./ncl set --help
usage: ncl set [-h] [--no-verify] [file [file ...]]

positional arguments:
  file         File containing desired state. stdin is used when no file is
               specified.

optional arguments:
  -h, --help   show this help message and exit
  --no-verify  Do not verify that the state was completely set and disable
               rollback to previous state.
```